spec/rails_helper.rb

```diff
@@ -12,7 +12,6 @@
 require "rails-controller-testing"
 require "rspec-activemodel-mocks"
 require "rspec/rails"
-require "webdrivers/chromedriver"
```

Comment:
Don't we need to require `selenium-webdriver` instead?

Reply:
No idea, there's no documentation, but it seems to work :)